nvflare/job_config/fed_job_config.py

```diff
@@ -230,7 +230,15 @@ def _copy_source_file(self, custom_dir, module, source_file, dest_file):
 
             import_source_file = os.path.join(source_dir, import_source.replace(".", os.sep) + ".py")
             if os.path.exists(import_source_file):
+                # Handle the import from within the same module
                 self._get_custom_file(custom_dir, import_module, import_source_file)
+            else:
+                # Handle the import from outside the module
+                size = len(module.split(".")) - 1
+                source_root = os.sep.join(source_dir.split(os.sep)[0:-size])
+                import_source_file = os.path.join(source_root, import_source.replace(".", os.sep) + ".py")
+                if os.path.exists(import_source_file):
```

Comment:
What happens if import_source_file does not exist? Is this an error or a normal case that does not require the source file to be copied over?

Reply:
If import_source_file does not exist, it simply skip and not copy to the job custom folder. That applies to the standard package, like "torch", etc.